README.md

```diff
@@ -99,76 +99,95 @@ If neither `RB` nor `RT` is pressed, the robot operates in _regular_ driving mod
 
 ---
 
-## Running on Panther robot
+## Docker image
+
+[![Build/Publish Docker Image](https://github.com/husarion/kortex_joystick_demo/actions/workflows/build-docker-image.yaml/badge.svg)](https://github.com/husarion/kortex_joystick_demo/actions/workflows/build-docker-image.yaml)
 
-1. Access NUC via ssh
+| ROS distro | Supported architectures |
+| :--------: | :---------------------: |
+|  `noetic`  |      `linux/amd64`      |
+
+Available on [Docker Hub](https://hub.docker.com/r/husarion/kortex-joystick/tags)
+
+### Demo
+
+#### Running on Panthers User Computer
+
+1. Access User Computer via ssh
 
    ```bash
    ssh husarion@10.15.20.3
    ```
 
    The default password is `husarion`
 
-2. Clone this repo
+2. Clone this repo on your Panther:
 
    ```bash
-   cd ~/husarion_ws/src/
    git clone https://github.com/husarion/kortex_joystick_demo.git
    ```
 
-3. Setup a gamepad according to [Setup joy](#setup-joy)
-
-4. Setup virtual desktop
+3. If you are using Kinova Gen3 Lite, edit [compose.kinova.yaml](./demo/compose.kinova.yaml) file and add `arm:=gen3_lite` launch parameter to the command
```

Comment:
Maybe instead we can use `.env` file, like it is used in other repositories, e.g. [rosbot-xl-manipualtion](https://github.com/husarion/rosbot-xl-manipulation/blob/main/.env) and define version there